.github/workflows/test.yaml

```
@@ -34,7 +34,7 @@ permissions:
 jobs:
   test:
     name: ${{ matrix.type }} ${{ matrix.flavor }}
-    runs-on: ubuntu-latest
+    runs-on: 'ubuntu-latest'
```

```suggestion
    runs-on: ubuntu-latest
```
Normally wouldn't mind but since there was the other change wanted to clean this up too